Makefile

```
@@ -0,0 +1,3 @@
+localtest: 
+	docker-compose up -d
+	KAFKA_SKIP_NETTEST=1 KAFKA_VERSION=2.3.1 	go test -race -cover ./...
```

I would personally split up running docker with running the tests, maybe something like
```suggestion
docker: 
	docker-compose up -d
	
test: 
	KAFKA_VERSION=2.3.1 \
		KAFKA_SKIP_NETTEST=1 \
		go test -race ./...
```

---

I've separated the steps. But I want test to be the default and to have a dependency on the docker step so changed your suggestion a little.

---

I don't think we need to be running docker every time, I was thinking it is more common to run `make docker` once, and run `make test` repeatedly?

Also it might've been missed, but I had a suggestion for the formatting of the test as well
```
	KAFKA_VERSION=2.3.1 \
			KAFKA_SKIP_NETTEST=1 \
			go test -race ./...
```

---

I've made the formatting change. 

Running docker compose takes a half second once the kafka cluster is already running. I'd like new users to have a low barrier to entry and consider the time worth it.

---

I know this nitpicky but I think `make docker test` is as low barrier to entry as `make test`

I think the distinction provides value because it may be confusing to someone who is not familiar with docker or docker compose when kafka containers suddenly start running on their laptop or if a `docker-compose` command fails

---

Done